packages/preview/cambridge-polylux-theme/0.0.1/example/main.typ

```diff
@@ -0,0 +1,60 @@
+#import "@preview/polylux:0.4.0": toolbox
+#import "../lib.typ": camDarkBlue, camLightBlue, slide
```

Comment:
I think it would make sense to update this import to use `@preview/…`, and make the `example` directory [a template](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#templates). This way, people will be able to start a new slideshow using the template easily both from the CLI and the web app.